docs/technical/01_addresses.md

```diff
@@ -2,174 +2,134 @@
-| Policy    | [`0x3dC18017cf8d8F4219dB7A8B93315fEC2d15B8a7`](https://goerli.etherscan.io/address/0x3dC18017cf8d8F4219dB7A8B93315fEC2d15B8a7) |
-| Emergency | [`0x3dC18017cf8d8F4219dB7A8B93315fEC2d15B8a7`](https://goerli.etherscan.io/address/0x3dC18017cf8d8F4219dB7A8B93315fEC2d15B8a7) |
-
-## Periphery Contracts
-
-### Flex Loans
+### Kernel
+
+|   Contract   |   Chain   | Address |
+|-----------|-----------------|-----------------|
+|   Kernel  |   Mainnet        | [`0x2286d7f9639e8158FaD1169e76d1FbC38247f54b`](https://etherscan.io/address/0x2286d7f9639e8158FaD1169e76d1FbC38247f54b) |
+|           |   Arbitrum       | [`0xeac3eC0CC130f4826715187805d1B50e861F2DaC`](https://arbiscan.io/address/0xeac3eC0CC130f4826715187805d1B50e861F2DaC) |
+|           |   Goerli       | [`0xDb7cf68154bd422dF5196D90285ceA057786b4c3`](https://goerli.etherscan.io/address/0xDb7cf68154bd422dF5196D90285ceA057786b4c3) |
+
+
+### Modules
+| Contract | Chain | Address |
+| -------- | -------- | -------- |
+| TRSRY    | Mainnet | [`0xa8687A15D4BE32CC8F0a8a7B9704a4C3993D9613`](https://etherscan.io/address/0xa8687A15D4BE32CC8F0a8a7B9704a4C3993D9613) |
+|          | Goerli  | [`0xD8C59cFe5afbDB83D904E56D379028a2f6A07a2D`](https://goerli.etherscan.io/address/0xD8C59cFe5afbDB83D904E56D379028a2f6A07a2D) |
+| MINTR    | Mainnet  |  [`0xa90bFe53217da78D900749eb6Ef513ee5b6a491e`](https://etherscan.io/address/0xa90bFe53217da78D900749eb6Ef513ee5b6a491e) |
+|          | Arbitrum | [`0x8f6406eDbFA393e327822D4A08BcF15503570D87`](https://arbiscan.io/address/0x8f6406eDbFA393e327822D4A08BcF15503570D87) |
+|          | Goerli   | [`0xa192fFBF73858831a137DD098a706139Ca96AbD5`](https://goerli.etherscan.io/address/0xa192fFBF73858831a137DD098a706139Ca96AbD5) |
+| PRICE    | Mainnet | [`0xd6C4D723fdadCf0D171eF9A2a3Bfa870675b282f`](https://etherscan.io/address/0xd6C4D723fdadCf0D171eF9A2a3Bfa870675b282f) |
+|          | Goerli  | [`0xD9ace3Be2d80006EF4D90A2D35D861a5C9F98252`](https://goerli.etherscan.io/address/0xD9ace3Be2d80006EF4D90A2D35D861a5C9F98252) |
+| RANGE    | Mainnet | [`0xb212D9584cfc56EFf1117F412Fe0bBdc53673954`](https://etherscan.io/address/0xb212D9584cfc56EFf1117F412Fe0bBdc53673954) |
+|          | Goerli  | [`0x446f06f8Df7d5f627B073c6349b948B95c1f9185`](https://goerli.etherscan.io/address/0x446f06f8Df7d5f627B073c6349b948B95c1f9185) |
+| ROLES    | Mainnet  | [`0x6CAfd730Dc199Df73C16420C4fCAb18E3afbfA59`](https://etherscan.io/address/0x6CAfd730Dc199Df73C16420C4fCAb18E3afbfA59) |
+|          | Arbitrum | [`0xFF5F09D5efE13A9a424F30EC2e1af89D867834d6`](https://arbiscan.io/address/0xFF5F09D5efE13A9a424F30EC2e1af89D867834d6) |
+|          | Goerli   | [`0xe9a9d80CE3eE32FFf7279dce4c2962eC8098f71B`](https://goerli.etherscan.io/address/0xe9a9d80CE3eE32FFf7279dce4c2962eC8098f71B) |
+| BLREG    | Mainnet | [`0x6CAfd730Dc199Df73C16420C4fCAb18E3afbfA59`](https://etherscan.io/address/0x375E06C694B5E50aF8be8FB03495A612eA3e2275) |
```

Comment:
either roles or blreg on mainnet is a typo (one is 0x6ca, the other is 0x375)

Reply:
Looks like production had a typo. Pushed a fix to this repo.